charts/devtron/templates/dashboard.yaml

```diff
@@ -69,13 +69,21 @@ spec:
     spec:
       terminationGracePeriodSeconds: 30
       restartPolicy: Always
+      {{- if $.Values.components.dashboard.podSecurityContext }}
+      securityContext:
+{{ toYaml $.Values.components.dashboard.podSecurityContext | indent 8 }}
```

Comment:
Use values from global only as we need to refer the same in sub charts too like security chart